File: Java/백준/Bronze/30008. 준영이의 등급/준영이의 등급.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {	
		Scanner scan = new Scanner(System.in);
		
		int member = scan.nextInt(); 
		int num = scan.nextInt();
		int [] arr = new int[num];
		for(int i=0; i<arr.length; i++) {
			int A = scan.nextInt()*100/member;
			if(A < 5) arr[i] = 1;
			else if(A < 12) arr[i] = 2;
			else if(A < 24) arr[i] = 3;
			else if(A < 41) arr[i] = 4;
			else if(A < 61) arr[i] = 5;
			else if(A < 78) arr[i] = 6;
			else if(A < 90) arr[i] = 7;
			else if(A < 97) arr[i] = 8;
			else if(A < 101) arr[i] = 9;
			}
		for(int i=0; i<arr.length; i++) {
			System.out.print(arr[i]+" ");
		}
	}
}

File: Python/백준/Bronze/30008. 준영이의 등급/준영이의 등급.py
member, num = input().split()
member, num = int(member), int(num)

arr = list(map(int, input().split()))
arr2 = [0] * len(arr)

for i in range(len(arr)):
    A = arr[i] * 100 // member
    if(A < 5): arr2[i] = 1;
    elif(A < 12): arr2[i] = 2
    elif(A < 24): arr2[i] = 3
    elif(A < 41): arr2[i] = 4
    elif(A < 61): arr2[i] = 5
    elif(A < 78): arr2[i] = 6
    elif(A < 90): arr2[i] = 7
    elif(A < 97): arr2[i] = 8
    elif(A < 101): arr2[i] = 9

for i in arr2:
    print(i, end=' ')